Review reb2

rebplu opened the pull request.

rebplu committed:
Create archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

comments, questions to work further

rebplu committed:
Update human.ttl

rebplu committed:
Update human.ttl

rebplu committed:
Update image.ttl

rebplu committed:
Update information-carrier.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update human.ttl

rebplu committed:
Update language.ttl

Mundart = Dialekt (Mundart only related to the dialect of Swiss German)

rebplu reviewed (commented):
wrong place for my comment but can not delete it, see next comment

rebplu committed:
Update archive-record.ttl

hanscools commented on Nie-ontologies/Generic-ontologies/human.ttl:
```diff
@@ -109,22 +110,22 @@ human:Female
 	a rdfs:Class;
 	rdfs:label "human female"@en, "menschlich weiblich"@de;
-	rdfs:comment """Being of a human female sex."""@en;
+	rdfs:comment """Human being of a female sex."""@en;
```
Here the trait is meant, not the human.
I propose to add the humans of the different sexes having the respective trait.

rebplu replied:
Copied your changes made lately in the master to commit 2d516dc (so no conflicts anymore).

hanscools commented on Nie-ontologies/Generic-ontologies/human.ttl:
```diff
@@ -325,28 +326,28 @@ human:PerformingArtist
 	a rdfs:Class;
 	rdfs:label "performing artist"@en, "Kunstperformer"@de, "artiste de la performance"@fr;
-	rdfs:comment """Artist in a performing art."""@en;
+	rdfs:comment """Role of an artist in a performing art."""@en;
```
Artist is a role.
-> Role of a person creating art through performing.

rebplu replied:
Changed (also for the following) in commits 2d516dc and c0e70be .

hanscools commented on Nie-ontologies/Generic-ontologies/human.ttl:
```diff
@@ -325,28 +326,28 @@ human:PerformingArtist
 	a rdfs:Class;
 	rdfs:label "musician"@en, "Musiker"@de, "musicien ou musicienne"@fr;
-	rdfs:comment """."""@en;
+	rdfs:comment """Role of a musician."""@en;
```
Similar to previous.

hanscools commented on Nie-ontologies/Generic-ontologies/human.ttl:
```diff
@@ -325,28 +326,28 @@ human:PerformingArtist
 	a rdfs:Class;
 	rdfs:label "singer"@en, "Sänger"@de, "chanteur ou chanteuse"@fr;
-	rdfs:comment """."""@en;
+	rdfs:comment """Role of a singer."""@en;
```
Similar to previous.

hanscools commented on Nie-ontologies/Generic-ontologies/human.ttl:
```diff
@@ -355,7 +356,7 @@ human:Dancer
 	a rdfs:Class;
 	rdfs:label "dancer"@en, "Tänzer"@de, "danseur ou danseuse"@fr;
-	rdfs:comment """."""@en;
+	rdfs:comment """ Role of a dancer."""@en;
```
Similar to previous.

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+@prefix infocar: <http://e-editiones.ch/ontology/information-carrier#>.
+
+@base <http://e-editiones.ch/ontology/archive-record>.
+@prefix archive-record: <http://e-editiones.ch/ontology/archive-record#>.
```
What about the name: 'archiving'?
Similar to 'publishing' etc.

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	a rdfs:Class;
+	rdfs:label "archive record"@en, "Archivdatensatz"@de;
+	rdfs:comment """A record as an entry in an Archive Information System. This record can be f.e. one or more fonds, one or more dossiers, one or more documents, one or more parts of a document,..."""@en;
+	rdfs:subClassOf rico:Record, rico:Instantiation, creating:Creation, document:Document.
```
document:Document is abstracted from a carrier. Is this what you mean here?
Because rico:Record seems to be abstracted too.
On the other hand rico:Instantiation seems to be an information carrier.
Note: document:Document rdfs:subClassOf creating:Creation.

rebplu replied:
Changed in commit 2453fec .

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	a rdfs:Class;
+	rdfs:label "provenance"@en, "Provenienz"@de;
+	rdfs:comment """The agent (f.e. organization, person), which created the archive record originally for its own purposes."""@en;
+	rdfs:subClassOf agent:Agent, rico:Agent.
```
AFAIU Provenance in rico is a relation, e.g. between a record and an agent, dealing with origin description.
TBD

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	a rdfs:Class;
+	rdfs:label "volume"@en, "Band"@de;
+	rdfs:comment """Volume as type of the carrier resp. the archival material type. It is also a special book: One book of a serie of books. One serie is a set of books which are mostly created for the same purpose by the same provenance."""@en;
+	rdfs:subClassOf archive-record:Carrier, infocar:Volume, archive-record:book.
```
archive-record:book -> archive-record:Book

rebplu replied:
done

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	a rdfs:Class;
+	rdfs:label "paper file"@en, "Papierdossier"@de;
+	rdfs:comment """Paper file as type of the carrier resp. the archival material type."""@en;
+	rdfs:subClassOf archive-record:Carrier.
```
TBD

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	a rdfs:Class;
+	rdfs:label "carrier reference"@en, "Trägerreferenz"@de;
+	rdfs:comment """A reference in a carrier. This could be, f.e., a page in a volume."""@en;
+	rdfs:subClassOf document:Structure, concept:Identifier.
```
There is e.g. document:ContentStructureReference (SCO document:Structure, concept:Identifier), on an abstract level.
TBD

rebplu replied:
done

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	rdfs:comment """Relating a archive record to its physical or electronical carrier."""@en;
+	rdfs:domain archive-record:ArchiveRecord;
+	rdfs:range archive-record:Carrier;
+	rdfs:subPropertyOf rico:hasCarrierType.
```
I assume also: rdfs:subPropertyOf infocar:isOnCarrier.

rebplu replied:
done

hanscools commented on Nie-ontologies/Generic-ontologies/archive-record.ttl:
```diff
@@ -0,0 +1,128 @@
+	rdfs:comment """Relating a archive record to the agent, which originally created the archive record for its own purposes."""@en;
+	rdfs:domain archive-record:ArchiveRecord;
+	rdfs:range archive-record:Provenance;
+	rdfs:subPropertyOf rico:isProvenanceOf.
```
Actually: rico:hasProvenance owl:inverseOf rico:isProvenanceOf.
TBD

rebplu replied:
solved

rebplu committed:
Update languages.ttl

rebplu committed:
Update linguistics.ttl

rebplu committed:
Update logic.ttl

rebplu committed:
Update human.ttl

Updates resolving the conflict (and männlicher Mensch instead of männliche Mensch etc.)

rebplu committed:
Update organization.ttl

rebplu committed:
Update physical-entity.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update publishing.ttl

rebplu committed:
Update scholarly-editing.ttl

rebplu committed:
Update teaching.ttl

rebplu committed:
Create marriage.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update marriage.ttl

rebplu committed:
Update marriage.ttl

rebplu committed:
Update marriage.ttl

rebplu committed:
Update marriage.ttl

rebplu committed:
Update human.ttl

rebplu committed:
Update human.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update archive-record.ttl

rebplu committed:
Update marriage.ttl

rebplu committed:
Update human.ttl

German translations and review

rebplu commented:
There are now conflicts because of some little changes in the part that was changed in the master after this branch was created: See 0f8e1b3

Commit:
Merge branch 'master' into review_reb2
